freqtrade/exchange/common.py

```diff
@@ -34,6 +34,7 @@ def _get_logging_mixin():
 MAP_EXCHANGE_CHILDCLASS = {
     'binanceus': 'binance',
     'binanceje': 'binance',
+    'wavesexchange': 'waves',
```

Comment:
This mapping is pointless.

it's only there for binance since binance has 3 different ccxt exchange-names which map to the same exchange.
unless waves has 2 different ccxt names (hint: it doesn't) - please use ccxt naming to avoid unnecessary confusion.

Reply:
Thanks for clarifying

Reply:
I didn't really plan to work more on this so this might get some delays, however the download bug I'll push to CCXT today.

Reply:
> I didn't really plan to work more on this so this might get some delays

I'm not sure how to interpret that - i'm fine with leaving this open _if you plan on making it mergable_ in the foreseeable future - but there's no point if it's a "dump and run" PR - where the work would now fall onto maintainers to finish the PR - simply because i see no incentive to include an exchange with this little volume.

It's not in a merge-able state at the moment (due to the wrong naming - as well as merge conflicts) and the other comment that was pointed out.

As it then doesn't provide any more benefit other than "knows the precise amount of candles" - i'm not sure if this PR makes much sense for an exchange this small, as it'll still be usable with freqtrade as it's an exchange supported by ccxt, just without knowing the precise number of candles.

Reply:
Alright well you can close it. I'm sorry for the confusion. It wasn't meant as a dump & run PR, I just lost interest due to some reasons I rather not talk about.

In any case, I'll fix the download part on the CCXT side.

Reply:
That's perfectly fine, life/circumstances change.